lang/cs/Org.Apache.REEF.Examples.HelloREEF/App.config

```diff
@@ -18,7 +18,15 @@ specific language governing permissions and limitations
 under the License.
 -->
 <configuration>
-    <startup> 
-        <supportedRuntime version="v4.0" sku=".NETFramework,Version=v4.5" />
-    </startup>
+  <startup>
+    <supportedRuntime version="v4.0" sku=".NETFramework,Version=v4.5" />
+  </startup>
+  <runtime>
```

Comment:
We require all REEF users to do the same now, right? Is there a way to automate it?